### PR TITLE
365 feature emily api default database entries

### PR DESCRIPTION
**Pull request by AshtonStephens**
## Description

Adds both a script for populating the Emily API with some default values and adds some Makefile commands to perform those commands.

To run the emily API locally with the pre-populated database run the following:

```
make emily-integration-env-up-populated-database
```

To repopulate the database you can run the following:

```
make emily-populate-database
```

This change works by introducing two sub directories of the `./devenv/dynamodb/` directory:
- `./devenv/dynamodb/populated`
- `./devenv/dynamodb/dynamic`

The populated directory will contain the populated database and the dynamic one will expect to take changes and be used for dynamic testing.

## Type of Change

- New feature

## Does this introduce a breaking change?
No

### CODE REVIEW

**Review by setbern — APPROVED** [review]
LGTM

File: devenv/dynamodb/populated/shared-local-instance.db
Comment (djordon) [inline devenv/dynamodb/populated/shared-local-instance.db:1]: Is this supposed to be committed?

**Review by djordon — COMMENTED** [review]
Looks good, just had one question.

File: devenv/dynamodb/populated/shared-local-instance.db
Comment (AshtonStephens) [reply devenv/dynamodb/populated/shared-local-instance.db:1]: Yes

**Review by djordon — APPROVED** [review]
Looks good. I think all is needed a `cargo fmt`

**Pull request merged**